src/BuiltInTools/dotnet-watch/HotReload/BlazorWebAssemblyDeltaApplier.cs

```diff
@@ -76,6 +95,15 @@ async Task<ImmutableArray<string>> GetApplyUpdateCapabilitiesCoreAsync()
                     ArrayPool<byte>.Shared.Return(buffer);
                 }
             }
+
+            static string GetDefaultCapabilities(Version? targetFrameworkVersion)
+                => targetFrameworkVersion?.Major switch
```

Comment:
Do we need to make this TFM-dependent? It shouldn't matter what the application targets. We know we are running on .NET 8 runtime, no? Or is it possible for .NET 8 runtime to load older version of WASM?

Reply:
I did it this way to handle the case where a .NET 8 SDK is used with a project targeting an earlier target framework version. I didn't think the dotnet watch tool would be tied to the runtime version of the app - is it?

Reply:
The runtime is not determined by the TFM, but rather by global.json if present, right?
If a project builds against net50 it would still run on .NET 8 if it is launched from .NET 8 SDK.

Reply:
Hm, I thought the TFM determined the runtime, not the SDK. But I could be wrong. I believe [this](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json#:~:text=The%20global.json%20file%20allows%20you%20to%20define%20which%20.NET%20SDK%20version%20is%20used%20when%20you%20run%20.NET%20CLI%20commands.%20Selecting%20the%20.NET%20SDK%20version%20is%20independent%20from%20specifying%20the%20runtime%20version%20a%20project%20targets) suggests that it's the TFM that controls the runtime version?

Reply:
Never mind, you are right.